pom.xml

```diff
@@ -317,15 +317,11 @@
         <groupId>com.spotify.metrics</groupId>
         <artifactId>semantic-metrics-core</artifactId>
         <version>${semantic-metrics.version}</version>
-        <exclusions>
-          <exclusion>
-            <groupId>com.codahale.metrics</groupId>
-            <artifactId>*</artifactId>
-          </exclusion>
-          <exclusion>
```

Comment:
It was added by https://github.com/spotify/styx/pull/884/files#diff-9c5fb3d1b7e3b0f54bc5c4182965c4fe1f9023d449017cece3005d3f90e8e4d8R325